templates/testingLibrary/jest.config.js.eta

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  preset: '<%= it.expo ? "jest-expo" : "react-native" %>',
+  preset: 'jest-expo',
```

Comment:
With the removal of the conditional expression I believe this could be changed to a static `jest.config.js` file, removing the `.eta`